packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -138,17 +150,6 @@ function Field({
     }));
 
     setOptions(updatedOptions);
-    updateSelectText(updatedOptions);
-  };
-
-  const updateSelectText = (updatedOptions: SelectOption[]) => {
-    hookForm.setValue(
-      `${hookFieldNamespace}.selectText`,
-      updatedOptions
-        .filter((opt) => opt.value)
-        .map((opt) => opt.value)
-        .join("\n")
-    );
   };
 
```

Comment:
`updateSelectText` doesn't need to be done now as the source of truth would be `options` going forward